the-journal-of-egyptian-archaeology.csl

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>The Journal of Egyptian Archaeology</title>
+    <id>http://www.zotero.org/styles/the-journal-of-egyptian-archaeology</id>
+    <link href="http://www.zotero.org/styles/the-journal-of-egyptian-archaeology" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-the-history-of-collections" rel="template"/>
+    <link href="http://www.ees.ac.uk/publications/journal-egyptian-archaeology.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <issn>0075-4234</issn>
+    <eissn>0307-5133</eissn>
```

Comment:
@POBrien333, where did you find the "0075-4234" ISSN? It's associated with the journal somehow, but it's marked as wrong/invalid by several catalogues:

https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=0075-4234&report=xml&format=text

    <ISSN ValidYN="Y" IssnType="Print">0307-5133</ISSN>
    <ISSN ValidYN="N" IssnType="Undetermined">0075-4234</ISSN>
    <ISSNLinking ValidYN="Y">0307-5133</ISSNLinking>
>

http://library.wur.nl/WebQuery/clc/1847031
> ISSN: 0307-5133
> Wrong ISSN: 0075-4234

http://catalogue.nla.gov.au/Record/2269193
> ISSN: 0075-4234
> Invalid ISSN: 0307-5133